pages/[key]/index.tsx

```diff
@@ -104,6 +80,33 @@ const StartPage: NextPage = () => {
     } catch (err: unknown) {
       console.error({ err });
     }
+  };
+
+  useEffect(() => {
+    if (retry === '1') {
+      const localStorageUserData = localStorage.getItem(LOCALSTORAGE_USER_DATA_NAME);
+      if (!localStorageUserData) {
+        return;
+      }
+
+      const applicantProperties = JSON.parse(localStorageUserData) as ApplicantProperties;
+
+      if (!applicantProperties.dob || !applicantProperties.email || !applicantProperties.firstName || !applicantProperties.lastName) {
```

Comment:
If we're able to (if it's not a field name defined by OnFido), I'd prefer not to use abbreviations like "dob". See https://github.com/NEARFoundation/near-kyc/blob/c44fa7cb75f307a60900f17aef12b65e9351257d/types/ApplicantProperties.ts and https://github.com/NEARFoundation/near-kyc/pull/10/files#diff-06066658bada731114226fdb8eee079b437ac8d1888348aa566d324a7212897cR35

Let's call it something clearer, like birthDate.

Reply:
Sadly the dob is from Onfido: https://documentation.onfido.com/#applicants